src/global/reco/reco.cc

```diff
@@ -308,6 +313,25 @@ void InitPlugin(JApplication *app) {
         app
     ));
 
+    app->Add(new JOmniFactoryGeneratorT<CollectionCollector_factory<edm4eic::Cluster>>(
+      "BarrelClusters",
+      {
+        "HcalBarrelClusters",
+        "EcalBarrelScFiClusters",
```

Comment:
```suggestion
        "EcalBarrelClusters",
```

Reply:
Updated, and checked output locally.  I'll have time to start working on the detector benchmark tomorrow